packages/cxl-lumo-styles/scss/vaadin-tabs.scss

```diff
@@ -0,0 +1,20 @@
+@import "@conversionxl/cxl-ui/scss/utils/mixins";
+
+:host([theme~="institute-layout-tabs"]) [part="tabs"] {
```

Comment:
Prefix all `theme` attributes `cxl`.

I should be able to grep through codebase to find all our custom `theme` attributes.